docs/book/src/topics/multitenancy.md

```diff
@@ -2,10 +2,16 @@
 
 To enable single controller multi-tenancy, a different Identity can be added to the Azure Cluster that will be used as the Azure Identity when creating Azure resources related to that cluster.
 
-This is achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-identity) library.
+This had been achieved using the [aad-pod-identity](https://azure.github.io/aad-pod-identity) library and is dperecated now and [workload identity](https://azure.github.io/azure-workload-identity/docs/) is the next iteration of it.
+
+Note that `Service Principal With Client Password` and `Service Principal With Certificate` identitites are based on AAD pod identity which is decsribed in the below sections.
 
 ## Identity Types
 
+### Workload Identity
+
+Follow this [link](./workload-identity.md) for workload identity docs.
+
 ### Service Principal With Client Password
```

Comment:
```suggestion
### AAD Pod Identity using Service Principal With Client Password (Deprecated)
```

Reply:
I can't add comments on the lines below but should be the same for other sections

`### AAD Pod Identity using Service Principal With Certificate (Deprecated)`
`### AAD Pod Identity using User-Assigned Managed Identity (Deprecated)`

Reply:
Sure, but for the following:

> ### AAD Pod Identity using User-Assigned Managed Identity (Deprecated)

I don't think User assigned managed identity as anything to do with AAD pod identity.

Reply:
it does, it uses AAD pod identity too https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/azure/scope/identity.go#L169 

https://azure.github.io/aad-pod-identity/docs/concepts/azureidentity/#azureidentityspec

Reply:
Thanks.